docker/oso-clam-update/centos7/start.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -e
+
+# This is useful so we can debug containers running inside of OpenShift that are
+# failing to start properly.
+
+if [ "$OO_PAUSE_ON_START" = "true" ] ; then
+  echo
+  echo "This container's startup has been paused indefinitely because OO_PAUSE_ON_START has been set."
+  echo
+  while true; do
+    sleep 10    
+  done
+fi
+
+#echo "Running config playbook"
+cd /var/lib/clamav/playbooks
+ansible-playbook /var/lib/clamav/playbooks/config.yml
+
+# Keep running freshclam and clamav-unofficial-sigs every 12 hours
```

Comment:
This looks like the three steps could get out of step if any fetch step takes longer than expected.

Better, perhaps, is one run-in-loop loop that runs each of (freshclam; unofficial; push to bucket) in sequence.

Reply:
ops-run-in-loop is what we use in place of cron for non-root containers. We'll switch to cron once that's supported.

Reply:
I'm not concerned about using ops-run-in-loop itself.  I'm concerned at having multiple copies of it running in parallel in the background when there's a distinct requirement for steps 1 and 2 to finish before step 3 starts.